lib/client/fileUpload.js

```diff
@@ -25,6 +25,10 @@ Template.afFileUpload.onCreated(function () {
     return this.data.atts.collection;
   };
 
+  this.hasBootstrap = () => {
```

Comment:
Let's make it as `afFileUpload` template helper

Reply:
I am not sure if it is still available in the sub-Templates and the custom Templates

Reply:
> I am not sure if it is still available in the sub-Templates and the custom Templates

 1. It should be available
 2. If not, it can be solved by switching to parent context via `../`

WDYT?

Reply:
Ok, I will rewrite it and test it.

Reply:
Is removed